templates/RELEASE_NOTES_unified.htm.tmpl

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd">
+  <head></head>
+  <body>
+    <h1>Release Notes</h1>
+    <p>The following features were introduced or enhanced in the {{ toLower .Version }} Conjur OSS suite.</p>
+
+    <h2>Components</h2>
+    <p>These are the components that comprise the Conjur Open Source Suite with links to their GitHub releases:</p>
+    <p>
+      <ul>
+         {{- range .Components }}
+         <li><a href="https://github.com/{{ .Repo }}/releases/tag/{{ .ReleaseName }}">{{ .Repo }} {{ .ReleaseName }}</a> ({{ .ReleaseDate }})</li>
+         {{- end }}
+      </ul>
+    </p>
+
+    <h2>Upgrade Instructions</h2>
+    <p>Upgrade instructions are available for the following suite components:</p>
+    <p>
+      <ul>
+        {{- range .Components }}
+        {{- if .UpgradeURL }}
+        <li><a href="{{ .UpgradeURL }} ">{{ .Repo }}</a></li>
+        {{- end }}
+        {{- end }}
+      </ul>
+    </p>
+
+    <h2>What's New by Component</h2>
+    <p>
+      <ul>
+        {{ range .Components -}}
+          <!-- href target here does not work as it doesn't yet escape the spaces/slashes -->
+          <li><MadCap:xref href="#{{ .Repo }}">{{ .Repo }}</MadCap:xref></li>
+        {{- end }}
+      </ul>
+    </p>
+
+    {{ range .Components -}}
+    <!-- href target here does not work as it doesn't yet escape the spaces/slashes -->
+    <h3><a name="cyberark-{{ .Repo }}"></a>{{ .Repo }}</h3>
```

Comment:
the `name` here needs to be the same as what's in `<MadCap:xref href="#{{ .Repo }}">` above (eg: `{{.Repo}}`)

so if `Repo` is usually `cyberark/repo`
and you update the MadCap:xref to be ``<MadCap:xref href="#{{  slashToDash .Repo }}">`
then this line should also be `<h3><a name="{{ slashToDash .Repo }}"></a>{{ .Repo }}</h3>`

Reply:
State of internal header links is for sure needing improvement but it might open up rabbit holes so I split that work into #113.